lib/user/UserService.ts

```diff
@@ -32,6 +32,24 @@ class UserService extends FirebaseService {
     return setDoc(userDocRef, user);
   }
 
+  async updateName(name: string, userId: string, ) {
+    const userDocRef = this.getDocRef(userId);
+
+    return updateDoc(userDocRef, {name: name});
+  }
+
+  async updateAvatarUrl(avatarUrl: string, userId: string ) {
+    const userDocRef = this.getDocRef(userId);
+
+    return updateDoc(userDocRef, {avatarUrl: avatarUrl});
+  }
+
+  async updateEmail(email: string, userId: string) {
+    const userDocRef = this.getDocRef(userId);
+
+    return updateDoc(userDocRef, {email: email});
+  }
+
```

Comment:
you could just a `updateUser(userId: string, userAttributes: IUserAttributes)` instead of all these methods. The `updateDoc` firebase method updates any attributes you provide in the second arguments